sim/hunter/wyvern_strike.go

```diff
@@ -0,0 +1,94 @@
+package hunter
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+)
+
+func (hunter *Hunter) getWyvernStrikeConfig(rank int) core.SpellConfig {
+	spellId := [4]int32{0, 458436, 458481, 458482}[rank]
+	baseDamage := [4]float64{0, 3, 4, 6}[rank]
```

Comment:
What do you think about adding the `/100 * 8` up here instead of in the `OnSnapshot` so that the sim doesn't have to do that calculation on each snapshot?

Reply:
Yeah I think that makes sense I'll move the calculation

Reply:
Oh yeah I guess that's true. You could just do `dotAPCoef` to be short if you want